### ISSUE TITLE
Caffe version

### ISSUE THREAD

**Issue by jjn037** [ShuangLI59/person_search#66]
I have found your caffe is different from official version，and I couldn't find a way to finetune weight   which have be trained on imagenet for my model ,where there some ways to solve the problem?

**Comment by Cysu** [comment]
Yes, we forked from the official caffe. We actually provide an ImageNet pretrained model as linked in README. I'm not sure what kind of model you have. You may need to manually change all the prototxts in `models/psdb/resnet50/` in order to match your own model.

**Comment by jjn037** [comment]
input_shape {
  dim: 1
  dim: 3
  dim: 224
  dim: 224
}
layer {
  bottom: "data"
  top: "conv1"
  name: "conv1"
  type: "Convolution"
  convolution_param {
    num_output: 64
    kernel_size: 7
    pad: 3
    stride: 2
  }
}
layer {
  bottom: "conv1"
  top: "conv1"
  name: "bn_conv1"
  type: "BatchNorm"
  batch_norm_param {
    use_global_stats: true
  }
}
layer {
  bottom: "conv1"
  top: "conv1"
  name: "scale_conv1"
  type: "Scale"
  scale_param {
    bias_term: true
  }
}
layer {
  bottom: "conv1"
  top: "conv1"
  name: "conv1_relu"
  type: "ReLU"
}
layer {
  bottom: "conv1"
  top: "pool1"
  name: "pool1"
  type: "Pooling"
  pooling_param {
    kernel_size: 3
    stride: 2
    pool: MAX
  }
}
layer {
  bottom: "pool1"
  top: "res2a_branch1"
  name: "res2a_branch1"
  type: "Convolution"
  convolution_param {
    num_output: 256
    kernel_size: 1
    pad: 0
    stride: 1
    bias_term: false
  }

like this prototxt, I have to add some layer like BatchNorm???

**Comment by Cysu** [comment]
@jjn037 Yes, I'm afraid that you may need to change both the caffe and the prototxts.

**Comment by jjn037** [comment]
Thank you for your patience


原始邮件
发件人:Tong Xiaonotifications@github.com
收件人:ShuangLI59/person_searchperson_search@noreply.github.com
抄送:姜佳男jiangjianan037@126.com; Mentionmention@noreply.github.com
发送时间:2017年11月9日(周四) 02:06
主题:Re: [ShuangLI59/person_search] Caffe version (#66)


@jjn037 Yes, I'm afraid that you may need to change both the caffe and the prototxts.
